docs/use_cases/model_applications/short_range/MODEMultivar_fcstRRFS_obsGOES_MRMS_BrightnessTemp_Lightning.py

```diff
@@ -119,10 +119,10 @@
 # This use case does not use any Python Embedding.
 
```

Comment:
@lisagoodrich It looks like there is a missing Python Embedding section between the MET Configuration section and the User Scripting section.  Please update the file to include the Python Embedding section and state “This use case does not use Python embedding.”

Reply:
@jprestop Help.  This "Python Embedding" section is already in this file with this language.  I left it alone.

Reply:
@lisagoodrich Ah!  I see it now.  Thank you for catching that. The title was hidden and I missed the text. Apologies!